xgcm/test/test_metrics.py

```diff
@@ -190,18 +200,6 @@ def test_get_metric_with_conditions_01():
     xr.testing.assert_allclose(get_metric, expected_metric)
 
 
-# @pytest.mark.parametrize("periodic", [True, False])
```

Comment:
Should we leave the periodic parametrization in here?

Reply:
Hmm wouldn't a `False` condition require setting boundary values?

Reply:
Yeah you could just set `boundary='extend`, which will be ignored when periodic is true. Just an idea